spec/system/viewing_sidebar_mobile_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+describe "Viewing sidebar mobile", type: :system, js: true, mobile: true do
+  fab!(:user) { Fabricate(:user) }
+  let(:sidebar_dropdown) { PageObjects::Components::SidebarHeaderDropdown.new }
+  let(:composer) { PageObjects::Components::Composer.new }
+
+  before do
+    @original_disable_animation = Capybara.disable_animation
+    Capybara.disable_animation = true
+    sign_in(user)
+  end
+
+  after { Capybara.disable_animation = @original_disable_animation }
+
+  it "does not display the sidebar by default" do
+    visit("/latest")
+
+    expect(sidebar_dropdown).to be_hidden
+  end
+
+  it "does not display the keyboard shortcuts button" do
+    visit("/latest")
+
+    sidebar_dropdown.click
+
+    expect(sidebar_dropdown).to be_visible
+    expect(sidebar_dropdown).to have_no_keyboard_shortcuts_button
+  end
+
+  it "collapses the sidebar when clicking outside of it" do
+    visit("/latest")
+
+    sidebar_dropdown.click
+
+    expect(sidebar_dropdown).to be_visible
+
+    # Simulate clicking outside of the sidebar
+    page.execute_script("document.elementFromPoint(#{sidebar_dropdown.width + 10}, 1).click();")
```

Comment:
This is so cool!

Nit: do you think we should extract this into a `#click_outside` method?

Reply:
I tried that but it didn't work and I was spending too much time figuring out why

```
     Selenium::WebDriver::Error::MoveTargetOutOfBoundsError:
       move target out of bounds
         (Session info: headless chrome=108.0.5359.124)
```

Reply:
Just leaving a note, we figured this out. `Capybara.w3c_click_offset` is `true` by default which means the coord offset starts in the _middle_ of the element, not top left.